website_menu_by_user_status/models/website_menu.py

```diff
@@ -14,27 +14,28 @@ class WebsiteMenu(models.Model):
     user_logged = fields.Boolean(
         string="User Logged",
         default=True,
-        help=_("If checked, "
-               "the menu will be displayed when the user is logged "
-               "and give access.")
+        help="If checked, "
+        "the menu will be displayed when the user is logged "
```

Comment:
```suggestion
             "the menu will be displayed when the user is logged "
```

Align these lines please.

Reply:
pep8 is complaining if I indent this.

Reply:
website_menu_by_user_status/models/website_menu.py:18:13: E131 continuation line unaligned for hanging indent
website_menu_by_user_status/models/website_menu.py:26:13: E131 continuation line unaligned for hanging indent

Reply:
@moylop260 What do you think about this?

Reply:
it is correct.
```python
help="if checked, "
"the menu will be displayed..." 
```

Since that it is a continuation of string.

Another way to align it like Yajo is:
```python
help=("if checked, "
      "the menu will be displayed...")
```

Both are correct.

Reply:
Actually also in the second way, you do not need the `(` and `)'.

Reply:
@NL66278 Did you mean that is fine using?

```python
help = "if checked, "
       "the menu will be displayed..."
```

FYI you will have a syntax error for this case `IndentationError: unexpected indent`

Reply:
Yes it is. Because everything is in the field definition, python knows the second line is a continuation string.

Reply:
I doubt there's a Syntax error there:

```python
>>> print( 
...   "one" 
...         "two" 
...             "three" 
...      "four")                                                                                                                                                                       
onetwothreefour
```

Reply:
print is a method so it is using a block `(...)` where there is not a syntax error.

Reply:
Yes, it's the same here:

```python
>>> def mymethod(help): 
...     print(help)                                                                                                    

>>> mymethod( 
...     help="This is some " 
...          "long help", 
... )                                                                                                                  
This is some long help
```

Python doesn't care at all about white space when there are surrounding parenthesis.